pom.xml

```diff
@@ -8,5 +8,17 @@
     <artifactId>func-test</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>6.14.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>3.9.1</version>
+        </dependency>
+    </dependencies>
 
 </project>
```

Comment:
@OlenaKh Please add an empty line at the end of the file.

Reply:
@extsoft new commit to PR added